starter-code/javascript/main.js

```diff
@@ -1,60 +1,122 @@
-var chronometer = new Chronometer();
-var btnLeft     = document.getElementById('btnLeft');
-var btnRight    = document.getElementById('btnRight');
-var minDec      = document.getElementById('minDec');
-var minUni      = document.getElementById('minUni');
-var secDec      = document.getElementById('secDec');
-var secUni      = document.getElementById('secUni');
-var milDec      = document.getElementById('milDec');
-var milUni      = document.getElementById('milUni');
-
+var chrono = new Chronometer();
+
+var btnLeft = document.getElementById('btnLeft');
+var btnRight = document.getElementById('btnRight');
+var minDec = document.getElementById('minDec');
+var minUni = document.getElementById('minUni');
+var secDec = document.getElementById('secDec');
+var secUni = document.getElementById('secUni');
+var milDec = document.getElementById('milDec');
+var milUni = document.getElementById('milUni');
+
+function showTime() {
+    setInterval(printTime, 10)  // to call printTime function every 10 milisec
+}
```

Comment:
So you and Guido have the same solution, and although it is a solution, it is not ideal because now we can two `setInterval`s running concurrently.  You have two better options: first you could set the html value directly from the Chronometer.js file, but then your code would not be properly modularized.  Secondly, you could pass the `printTime` function as an argument to the `chrono.startClick` function.  So something like:
```
Chronometer.prototype.startClick = function(printTime) {
  this.intervalId = setInterval((function() {
    this.currentTime++
    this.setTime()
    printTime()
  }).bind(this), 1000);
};
```
This way, your data is defined and structured in one file, and your executable javascript is in another.